CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 ## Fixes
 - fix calling return type getter function `@Field(type => Foo)` before finishing module evaluation (allow for extending circular classes using `require`)
 - fix nullifying other custom method decorators - call the method on target instance, not the stored reference to original function (#247)
+- Prevent unnecessary conversion of a object that is already of the requested typ, to avoid constructor side-effects (#229)
```

Comment:
I usually refer to pull requests, not the original issues - so this should be #249.